web/src/pages/SignIn/index.tsx

```diff
@@ -51,10 +51,10 @@ const SignIn: React.FC = () => {
         return;
       } else {
         addToast({
-        type: 'error',
-        title: 'Erro na autenticação',
-        description: 'Ocorreu um erro ao fazer login. Cheque suas credenciais.'
-      });
+          type: 'error',
+          title: 'Erro na autenticação',
```

Comment:
"Verifique seu e-mail" may wrongly direct the user to check it's inbox . Just "Verifique suas credenciais" looks fine to me, since login user are also part of credentials, ana not only password

Reply:
Currently, the toast message is the same whether the credentials are incorrect or the user has not confirmed their email yet, thus the request.